haproxy.go

```diff
@@ -1,10 +1,33 @@
 package main
 
 import (
+	"encoding/csv"
+	"errors"
+	"fmt"
+	"log"
+	"net/http"
 	"strconv"
 	"strings"
 )
 
+const (
+	// This is the index of the process name for each of the
+	// processes currently being tracked by HaProxy
+	haProxyPxnameIndex = 0
+	// This is the index of the status of each process
+	// currently being tracked by HaProxy
+	haProxyStatusIndex = 17
+	// This is the status page that can be read when HaProxy is
+	// running
+	haProxyStatusPage = "http://localhost:%d/;csv"
+	// This is a name given to the app being deployed when setting
+	// it up in HaProxy. The name can be anything, it just
+	// needs to be consistent from when the server is first
+	// started, as it is used to define the currently active
+	// deploy
+	backendProcessName = "thing-"
```

Comment:
let's change it away from "thing" so (a) better name and (b) sanity check nothing else depends on "thing"